### ISSUE TITLE
Install fails due to script issue

### ISSUE THREAD

**Issue by Botspot** [ArchitektApx/zen-browser-copr#1]
I ran the script from your releases page and it failed to function properly.
```
pi@raspberrypi:~ $ sudo bash /home/pi/Downloads/install-zen-browser-arm64.sh
Creating Temp Directory in /tmp/zen-browser-install ...
Downloading Zen Browser Install Files ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 16629    0 16629    0     0  34894      0 --:--:-- --:--:-- --:--:-- 34894
Warning: Binary output can mess up your terminal. Use "--output -" to tell 
Warning: curl to output it to your terminal anyway, or consider "--output 
Warning: <FILE>" to save to a file.
Extracting Zen Browser Install Files ...
tar (child): zen.linux-generic.tar.bz2: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
Installing Zen Browser to /opt/zen-browser ...
cp: cannot stat 'zen/*': No such file or directory
install: cannot stat '/opt/zen-browser/browser/chrome/icons/default/default16.png': No such file or directory
install: cannot stat '/opt/zen-browser/browser/chrome/icons/default/default32.png': No such file or directory
install: cannot stat '/opt/zen-browser/browser/chrome/icons/default/default48.png': No such file or directory
install: cannot stat '/opt/zen-browser/browser/chrome/icons/default/default64.png': No such file or directory
install: cannot stat '/opt/zen-browser/browser/chrome/icons/default/default128.png': No such file or directory
Cleaning Up ...
Done! Zen Browser is installed.
You can now run Zen Browser by typing zen-browser in the terminal
or by searching for it in your applications menu.
pi@raspberrypi:~ $ zen-browser
/usr/bin/zen-browser: line 18: /opt/zen-browser/zen-bin: No such file or directory
```

Line 18 is the problem.
```
curl -L https://github.com/ArchitektApx/zen-browser-arm64-copr/releases/download/1.0.1-a.19/zen.linux-generic.tar.bz2
```
You forgot to put `-o zen.linux-generic.tar.bz2` at the end of that line.

**Comment by Botspot** [comment]
Also your hyperlink to the script on the README is broken.

**Comment by ArchitektApx** [comment]
fixed in 3d41b9e1d28b2f108666fc21903a3e99fa8e428e
